service/create/subnet.go

```diff
@@ -0,0 +1,69 @@
+package create
+
+import (
+	"fmt"
+
+	awsutil "github.com/giantswarm/aws-operator/client/aws"
+	awsresources "github.com/giantswarm/aws-operator/resources/aws"
+	"github.com/giantswarm/aws-operator/service/key"
+	"github.com/giantswarm/awstpr"
+	"github.com/giantswarm/microerror"
+)
+
```

Comment:
I extracted creating and deleting subnets to reduce the size of the processCluster and delete func methods.

Reply:
These kind of comments in the PRs are really useful (they explain the 'why' of the related changes), but will be lost after the PR is merged. Why not add them to source code?

Reply:
Generally I agree with this a lot . But in this case the reason for extracting it is because the processCluster and delete funcs are so huge. 

As part of the update story we need to rewrite the operator to use operatorkit and cloud formation. So I don't think including this in the comments is useful at this point.